KaggleTorrent/main.py

```diff
@@ -57,13 +63,47 @@ def main():
     db_engine = db_connection_handler.create_sqlalchemy_engine()
     print("Connected to the database")
 
-    if command == 'generate':
-
+    # CHECK USER VARIABLES
+    proceed = None
+
+    # Check db emptiness
+    if database_exists(db_engine.url):
+        if command == 'init':
+            print(f'Database {config.db_name} already exist. Provide database name that does not exist.',
+                  file=sys.stderr)
+            proceed = False
+        if command == 'refresh':
+            print(f'Database {config.db_name} already exist. This operation will reinitialize the current database')
+            print('and populate it with provided MetaKaggle version.')
+            ans = input(f'Are you sure to re-initialize {config.db_name} database? [yes]\n')
```

Comment:
Same typo here, "Database [...] already exist**s**".

"[...] and populate it with **the** provided Meta Kaggle version."